src/partitioned-dbs/index.rst

```diff
@@ -105,14 +105,20 @@ sensor readings from an IoT field monitoring service. In this particular
 use case, it's quite logical to group all documents by their ``sensor_id``
 field. In this case, we would call the ``sensor_id`` the partition.
 
+A partition offers two benefits. Firstly, it allows you to simplify the
+workload on the query engine, by allowing the engine to only query shards
+that it needs to query. Secondly, it allows the database to scale in more
+predictable ways.
+
 A good partition has two basic properties. First, it should have a high
 cardinality. That is, a large partitioned database should have many more
-partitions than documents in any single partition. A database that has
-a single partition would be an anti-pattern for this feature. Secondly,
-the amount of data per partition should be "small". The general
-recommendation is to limit individual partitions to less than ten
-gigabytes (10 GB) of data. Which, for the example of sensor documents,
-equates to roughly 60,000 years of data.
+partitions than documents, compared to an unpartitioned database. A database
+that has a single partition would be an anti-pattern for this feature, as
+one of the major bonuses of partitioning is that it allows index lookups to
+iterate over a smaller B-Tree. Secondly, the amount of data per partition
```

Comment:
Would prefer the discussion didn't get into the implementation detail that, in CouchDB 3.x, storage is B-Tree based. This won't be the case with FoundationDB (CouchDB 4.0). It'd be sufficient to say that "it allows index lookups to occur over a small set of documents."